asciidoctorj-core/src/test/groovy/org/asciidoctor/extension/GithubContributorsBlockMacro.groovy

```diff
@@ -0,0 +1,45 @@
+package org.asciidoctor.extension
+
+import groovy.json.JsonSlurper
+import org.asciidoctor.ast.AbstractBlock
+import org.asciidoctor.ast.Column
+import org.asciidoctor.ast.Row
+import org.asciidoctor.ast.Table
+
+class GithubContributorsBlockMacro extends BlockMacroProcessor {
+
+    GithubContributorsBlockMacro(String macroName) {
+        super(macroName)
+    }
+
+    @Override
+    Object process(AbstractBlock parent, String target, Map<String, Object> attributes) {
+        String content = new URL("https://api.github.com/repos/${target}/contributors").text
```

Comment:
For the purpose of the test, I recommend using local date for this extension. You can activate the real data for the version in the asciidoctorj-extensions-lab.